src/app/containers/StoryPromo/MediaIndicator/index.jsx

```diff
@@ -2,22 +2,45 @@ import React from 'react';
 import moment from 'moment-timezone';
 import { shape, bool, string } from 'prop-types';
 import MediaIndicatorComp from '@bbc/psammead-media-indicator';
+import path from 'ramda/src/path';
 import pathOr from 'ramda/src/pathOr';
 import pick from 'ramda/src/pick';
 import { storyItem } from '../../../models/propTypes/storyItem';
 import formatDuration from '../../../lib/utilities/formatDuration';
 
-const MediaIndicator = ({ item, topStory, service, indexAlsos }) => {
-  const isPGL = pathOr(null, ['cpsType'], item) === 'PGL';
-  const isMedia = pathOr(null, ['cpsType'], item) === 'MAP';
-  const hasMediaInfo = pathOr(null, ['media'], item);
+const getAssetContentTypes = item => {
+  const mediaContentTypes = ['video'];
+  let type = pathOr(null, ['contentType'], item);
+  if (type) {
+    type = type.toLowerCase();
+  }
+  return mediaContentTypes.includes(type) ? type : null;
```

Comment:
Here `type` could still be null. Better do a null check before this.

Reply:
But if the type is null then mediaContentTypes.includes(type) will be false and it will return null. Isn't that correct?

Reply:
I think it's because you're using `inlcudes()` which would check if `null` is in `mediaContentTypes` which would return `true`.

Reply:
Thats correct, but i would avoid doing an` includes` check on null value. This is what i was thinking
```js
if(!type) { // we first do a falsy check
  return null; // return early if type is falsy
}
type = type.toLowerCase();
return mediaContentTypes.includes(type) ? type : null;
```

Reply:
Okay, got it, thank you @HarveyPeachey  @sadickisaac  !

Reply:
Couldn't you just change it to something like this? 
```
const getAssetContentTypes = item => {
  const mediaContentTypes = ['video'];
  const type = pathOr(null, ['contentType'], item).toLowerCase();
  return mediaContentTypes[0] === type ? type : null;
};
```
Or would that not work?

Reply:
If we have two `mediaContentTypes` that would break. @HarveyPeachey

Reply:
How about this?

```
const getAssetContentTypes = item => {
  const mediaContentTypes = ['video'];
  const type = pathOr('', ['contentType'], item).toLowerCase();
  return mediaContentTypes.includes(type) ? type : null;
};
```

Reply:
This also wouldn't work because the `MediaIndicator` is to be rendered only when the media type is known. `''` is not the same as `null`